### PR TITLE
feat(TDKN-229): Add more tql client operators

### PR DESCRIPTION
**Pull request by jdescottes-talend**
**What is the problem this Pull Request is trying to solve?**
`in`, `lessThanOrEqual`, `greaterThanOrEqual`, `unequal` operators not existing in the client lib.

**What is the chosen solution to this problem?**
Add the implementation for each of them.

**Link to the JIRA issue**

https://jira.talendforge.org/browse/TDKN-229
 
**Please check if the Pull Request fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/Talend/daikon/blob/master/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features, coverage should be over 75% in the new code)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in JIRA), if any, are all linked or available in the Pull Request

 
**[ ] This Pull Request introduces a breaking change**

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/204/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/204/)

**Comment by jdescottes-talend** [comment]
> Can you please elaborate why you need a `unequal` operator when `not` and `equals` exists?

First of all, the operator already exists in the lib core :
https://github.com/Talend/daikon/blob/master/daikon-tql/daikon-tql-core/src/main/antlr4/TqlLexer.g4

Then, it's easier to use (more straightforward) and not circumvoluted.

Moreover, we can argue the same for the other operators :
- `field in [1, 2, 3]` <=> `(field = 1) or (field = 2) or (field = 3)`
- `field <= 36` <=> `not (field > 36)`
- `field >= 42` <=> `not (field < 42)`

Without all those specific operators, the client lib isn't directly usable for the previous cases and custom wrapped code would be needed.

File: daikon-tql/daikon-tql-client/src/converter/operators/in.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import Operator, {
+	isDefined,
+	wrap,
+} from './operator';
+
+/**
+ * Class representing the In operator.
+ * Will be serialized as follows : (field1 in [42, 76])
+ */
+export default class Between extends Operator {
```
Comment (ncomont) [inline daikon-tql/daikon-tql-client/src/converter/operators/in.js]: I guess you mean `In`

Comment (jdescottes-talend) [reply daikon-tql/daikon-tql-client/src/converter/operators/in.js]: Good catch ! :)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/207/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/207/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/208/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/208/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/213/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/213/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/214/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/214/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/215/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/215/)

**Comment by fhuaulme** [comment]
Need a TDKN Jira in the commit message for change tracking.

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/217/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/217/)

**Pull request merged**